plsync/mlabconfig_test.py

```diff
@@ -426,6 +426,57 @@ def create_new_fake_file(*args):
             self.assertEqual(contents.strip(),
                              virtual_output_files[fname].getvalue().strip())
 
+    def test_export_legacy_includes_all_experiments(self):
+        # Setup synthetic user, site, and experiment configuration data.
+        experiments = [model.Slice(name='abc_bar',
+                                   index=1,
+                                   attrs=self.attrs,
+                                   users=self.users,
+                                   use_initscript=True,
+                                   ipv6='all')]
+        # Assign experiments to nodes.
+        for hostname, node in self.sites[0]['nodes'].iteritems():
+            experiments[0].add_node_address(node)
+        output = StringIO.StringIO()
+        expected_config = {
+            u'labels': {u'service': u'sidestream'},
+            u'targets': [
+                u'bar.abc.mlab2.abc01.measurement-lab.org:9090',
+                u'bar.abc.mlab1.abc01.measurement-lab.org:9090',
+                u'bar.abc.mlab3.abc01.measurement-lab.org:9090']
```

Comment:
Formatting question: should the closing square bracket go on the following line, aligned with `u'targets':`?